internal/cli/kraft/run/run.go

```diff
@@ -368,7 +370,16 @@ func (opts *RunOptions) Run(ctx context.Context, args []string) error {
 
 	var exitErr error
 	if !opts.Detach {
-		exitErr = logs.FollowLogs(ctx, machine, opts.machineController)
+		if opts.Prefix == "name" {
+			opts.Prefix = machine.Name
+		}
```

Comment:
Prefer to use dedicated flags rather than create special values. What if the service is called `name`?

Instead, please introduce a `--prefix-name` flag which does this functionality. And prefer the value of `--prefix` if it is set.

Reply:
Added